source/playbooks/NIST80053/lib/security_controls_playbook-construct.ts

```diff
@@ -0,0 +1,133 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { Stack, CfnMapping, App, StackProps, CfnParameter, Aspects } from 'aws-cdk-lib';
+import { StringParameter } from 'aws-cdk-lib/aws-ssm';
+import { Trigger } from '../../../lib/ssmplaybook';
+import { Construct } from 'constructs';
+import { ControlRunbooks } from './control_runbooks-construct';
+import AdminAccountParam from '../../../lib/admin-account-param';
+import { Runtime } from 'aws-cdk-lib/aws-lambda';
+import { IControl } from '../../../lib/sharrplaybook-construct';
+import { WaitProvider } from '../../../lib/wait-provider';
+import SsmDocRateLimit from '../../../lib/ssm-doc-rate-limit';
+
+export interface SecurityControlsPlaybookProps extends StackProps {
+  solutionId: string;
+  solutionVersion: string;
+  solutionDistBucket: string;
+  solutionDistName: string;
+  remediations: IControl[];
+  securityStandard: string;
+  securityStandardLongName: string;
+  securityStandardVersion: string;
+}
+
+export class SecurityControlsPlaybookPrimaryStack extends Stack {
```

Comment:
This shouldn't be necessary, you should be able to use the "PlaybookPrimaryStack" or whatever the admin stack is called that the other standards use. SC is the special case.